src/interfaces/store-interfaces/state-interfaces/dissolution-state-interface.ts

```diff
@@ -1,10 +1,13 @@
-import { DissolutionTypes } from '@/enums'
-import { DissolutionStatementIF, OrgPersonIF } from '@/interfaces'
+import { DissolutionStatementTypes, DissolutionTypes } from '@/enums'
+import { OrgPersonIF } from '@/interfaces'
 
 export interface DissolutionStateIF {
   dissolutionType: DissolutionTypes
   dissolutionDate: string
-  dissolutionStatementStep: DissolutionStatementIF
+  dissolutionStatementStep: {
+    valid: boolean
+    dissolutionStatementType?: DissolutionStatementTypes
+  },
```

Comment:
I removed a nested interface here.

In my opinion, a nested interface is mostly useful if you're going to use the property as a whole. Here, we don't need to do that, so I simplified things.